### PR TITLE
reviewed icos for a bunch of integrations

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary

Reviewed icons for integrations across these categories
image

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by ilyam8 — APPROVED** [review]

**Review by andrewm4894 — APPROVED** [review]

File: integrations/deploy.yaml [hunk 1]
```diff
@@ -188,7 +188,7 @@
     link: ''
     categories:
       - deploy.operating-systems
-    icon_filename: 'apple.png'
+    icon_filename: 'apple.svg'
```
Comment (ilyam8) [inline integrations/deploy.yaml:191]: the current is `macos.svg`

**Review by sashwathn — DISMISSED** [review]

File: integrations/deploy.yaml (hunk 1 above)
Comment (hugovalente-pm) [reply integrations/deploy.yaml:191]: I think the MacOS one doesn't seem right
image

We have 3 icons for Mac/apple, the othe two the black one is a png and the third is the svg (not added by me was already there)
image

image

so I would go with the grey one and we will need to fix the `macos.svg` to `apple.svg`

cc/ @sashwathn @shyamvalsan

Comment (hugovalente-pm) [reply integrations/deploy.yaml:191]: > This PR is [28 commits behind](https://github.com/netdata/netdata/compare/hugo-fix-data-collection-icons-1...master) master.

@ilyam8 I think this is fixed, right?

**Review by shyamvalsan — APPROVED** [review]

File: integrations/deploy.yaml (hunk 1 above)
Comment (Ancairon) [reply integrations/deploy.yaml:191]: looks like it, now it includes only the yamls, not past commits

Comment (Ancairon) [reply integrations/deploy.yaml:191]: (removing the guys that were dragged in here due to no rebase)

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**